SharpAdbClient/AdbClient.cs

```diff
@@ -344,33 +344,23 @@ public IEnumerable<ForwardData> ListForward(DeviceData device)
         }
 
         /// <inheritdoc/>
-        public async Task<Image> GetFrameBufferAsync(DeviceData device, CancellationToken cancellationToken)
+        public Framebuffer GetRefreshableFramebuffer(DeviceData device)
         {
             this.EnsureDevice(device);
 
-            using (IAdbSocket socket = Factories.AdbSocketFactory(this.EndPoint))
-            {
-                // Select the target device
-                this.SetDevice(socket, device);
-
-                // Send the framebuffer command
-                socket.SendAdbRequest("framebuffer:");
-                socket.ReadAdbResponse();
-
-                // The result first is a FramebufferHeader object,
-                var size = Marshal.SizeOf(typeof(FramebufferHeader));
-                var headerData = new byte[size];
-                await socket.ReadAsync(headerData, cancellationToken).ConfigureAwait(false);
+            return new Framebuffer(device, this);
+        }
 
-                var header = FramebufferHeader.Read(headerData);
+        /// <inheritdoc/>
+        public async Task<Image> GetFrameBufferAsync(DeviceData device, CancellationToken cancellationToken)
```

Comment:
@qmfrederik would it not be better to call the method CreateRefreshableFramebuffer?